examples/main/README.md

```diff
@@ -208,6 +208,14 @@ Top-p sampling, also known as nucleus sampling, is another text generation metho
 
 Example usage: `--top-p 0.95`
 
+### Min P Sampling
+
+-   `--min-p N`: Sets a minimum base probability threshold for token selection (default: 0.0).
+
+Min-P sampling is a sampling method where the value represents the base required probability percentage. This value is scaled down based on the probability of the most likely token. For example, with a Min-P value set at 0.05 and the highest token probability being 90%, the percentage requirement threshold becomes 4.5%. This approach ensures a balance of quality and variety in the results, and was designed as an alternative to improve upon Top P. The default value is 0.05.
```

Comment:
it's not actually "percentage" + I don't think the word "base" adds anything beside padding. (first sentence only, after that the word "percentage" refers to example values)

Reply:
Base was added to distingiush between the scaled probability vs the base value. But it might be implied well enough already